inference/enterprise/device_manager/helpers.py

```diff
@@ -1,10 +1,15 @@
 import time
+import uuid
 
 from inference.core.cache import cache
 from inference.core.env import METRICS_INTERVAL
 
 
-def get_cache_model_items():
+def get_device_id():
+    return hex(uuid.getnode())
```

Comment:
cc @SolomonLake this is how I'm getting the device ID. `platform.node()` doesn't work for hyperkit based docker engines (might just be a macOS thing). I'm open to using platform.node though, just want to keep things consistent

Reply:
I think that works just fine! Any unique id for the container works well.